suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -270,13 +393,18 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
         const supportedNetworksSymbols = supportedNetworks.map(network => network.symbol);
         const discoveryNetworksTotalCount = supportedNetworksSymbols.length;
 
-        let availableCardanoDerivationsResponse;
+        let availableCardanoDerivations: ('normal' | 'legacy' | 'ledger')[] | undefined;
```

Comment:
Please extract this type from `getAvailableCardanoDerivationsThunk` to reuse it.